### PR TITLE
Various fixes after tests

### PR DESCRIPTION
**Pull request by philogicae**
While testing, to avoid creating too many PRs and by convenience, I increment all issues found & fixed here:
- [Ensure crn_node urls without ending slash](https://github.com/aleph-im/aleph-client/commit/10c37c71a0946007dcf98ea5d80ef3bffbee8186)

- [Fix wrong descriptions on "instance create --help" cmd](https://github.com/aleph-im/aleph-client/commit/0be87c1fcb1183d9e6f40ee7dc7f4f53dc681c8c)

- [Skip rootfs prompting if valid hash is provided via --rootfs arg](https://github.com/aleph-im/aleph-client/commit/0410f100fa877736df8982e9774172364947e946)

- [Improve printed messages](https://github.com/aleph-im/aleph-client/commit/eb31f6a7051d5387dcfce9a0fd3657ee3a8c27e2)

- [Needed fix for hold-tier instances (payment=hold & no receiver)](https://github.com/aleph-im/aleph-client/commit/fe21550df9365d7e865fd4209d8dd4c2ff6890d0)

- [Fix cmd "instance list": ipv6 not found/displayed properly](https://github.com/aleph-im/aleph-client/pull/243/commits/4d50c7e2dc00bf9cbc36f7977889cd7248ff8bf8)

- [Improve infos/display on cmd: "instance list"](https://github.com/aleph-im/aleph-client/pull/243/commits/20ec332e922296e4e544066c9ab01dd16ddf5bc9)

- [Fix aleph-message 0.4.8, change default values for ram/hdd, fixes for review](https://github.com/aleph-im/aleph-client/pull/243/commits/e5284aee6639002779c3cf2fa0f9f142b3871a57)

### CODE REVIEW

**Review by Psycojoker — APPROVED** [review]
LGTM from a code point of view

File: src/aleph_client/commands/instance/__init__.py [hunk 1]
```diff
@@ -353,57 +355,137 @@ async def delete(
...
-                    )
+                    path = f"{node['address'].rstrip('/')}/about/executions/list"
                     # Fetch from the CRN API if payment
                     executions = await fetch_json(session, path)
                     if message.item_hash in executions:
-                        ipv6_address = executions[message.item_hash]["networking"]["ipv6"]
-                        return message.item_hash, ipv6_address
-
-            return message.item_hash, "Not available (yet)"
+                        details["ipv6_logs"] = executions[message.item_hash]["networking"]["ipv6"].split("/")[0]
```
Comment (hoh) [inline src/aleph_client/commands/instance/__init__.py:392]: Would this make it clearer than the split on string ? (import `ipaddress` from builtins)

```suggestion
                        interface = ipaddress.IPv6Interface(executions[message.item_hash]["networking"]["ipv6"]
                        details["ipv6_logs"] = str(interface).ip)
```

File: src/aleph_client/commands/instance/__init__.py [hunk 2]
```diff
@@ -354,62 +355,137 @@ async def delete(
         typer.echo(f"Instance {item_hash} has been deleted. It will be removed by the scheduler in a few minutes.")
 
 
-async def _get_ipv6_address(message: InstanceMessage, node_list: NodeInfo) -> Tuple[str, str]:
+async def _get_instance_details(message: InstanceMessage, node_list: NodeInfo) -> Tuple[str, str]:
     async with ClientSession() as session:
-        is_hold = not message.content.payment or message.content.payment.type == PaymentType["hold"]
-        is_confidential = (
+        hold = not message.content.payment or message.content.payment.type == PaymentType["hold"]
+        confidential = (
```
Comment (hoh) [inline src/aleph_client/commands/instance/__init__.py:361]: Why rename `is_hold` -> `hold` and `is_confidential` -> `confidential` ?

**Review by hoh — COMMENTED** [review]
I added a few comments.

Looks like the test fail due to typing inconsistencies:
```
 src/aleph_client/commands/instance/__init__.py:365:21: error: Item "None" of "Optional[TrustedExecutionEnvironment]" has no attribute "firmware"  [union-attr]
src/aleph_client/commands/instance/__init__.py:365:21: error: Argument 1 to "len" has incompatible type "Union[ItemHash, None, Any]"; expected "Sized"  [arg-type]
src/aleph_client/commands/instance/__init__.py:369:46: error: Item "None" of "Optional[Payment]" has no attribute "type"  [union-attr]
src/aleph_client/commands/instance/__init__.py:380:24: error: Incompatible return value type (got "Tuple[Any, Dict[str, Union[bool, str, PaymentType, Any, None]]]", expected "Tuple[str, str]")  [return-value]
src/aleph_client/commands/instance/__init__.py:382:45: error: Item "None" of "Optional[Payment]" has no attribute "receiver"  [union-attr]
src/aleph_client/commands/instance/__init__.py:385:41: error: Item "None" of "Optional[HostRequirements]" has no attribute "node"  [union-attr]
src/aleph_client/commands/instance/__init__.py:385:41: error: Item "None" of "Union[NodeRequirements, None, Any]" has no attribute "node_hash"  [union-attr]
src/aleph_client/commands/instance/__init__.py:393:32: error: Incompatible return value type (got "Tuple[ItemHash, Dict[str, Union[bool, str, PaymentType, Any, None]]]", expected "Tuple[str, str]")  [return-value]
src/aleph_client/commands/instance/__init__.py:397:16: error: Incompatible return value type (got "Tuple[ItemHash, Dict[str, Union[bool, str, PaymentType, Any, None]]]", expected "Tuple[str, str]")  [return-value]
src/aleph_client/commands/instance/__init__.py:412:17: error: Invalid index type "str" for "str"; expected type "Union[SupportsIndex, slice]"  [index]
src/aleph_client/commands/instance/__init__.py:430:23: error: Invalid index type "str" for "str"; expected type "Union[SupportsIndex, slice]"  [index]
src/aleph_client/commands/instance/__init__.py:434:21: error: Invalid index type "str" for "str"; expected type "Union[SupportsIndex, slice]"  [index]
src/aleph_client/commands/instance/__init__.py:448:18: error: Invalid index type "str" for "str"; expected type "Union[SupportsIndex, slice]"  [index]
```

**Pull request merged**